content/download.html

```diff
@@ -121,6 +121,8 @@ <h3>Community-maintained packages</h3>
     <dd>
       Stable releases are available from <a href="https://flathub.org/apps/details/net.openra.OpenRA">Flathub</a>:
       <pre>$ flatpak install --from https://flathub.org/repo/appstream/net.openra.OpenRA.flatpakref</pre>
+      Playtests are available from <a href="https://flathub.org/beta-repo/appstream/net.openra.OpenRA.flatpakref">Flathub Beta</a>:
```

Comment:
Can we please make this something like
```html
<dd>
  Stable releases and Playtests are available from <a href="https://flathub.org/apps/details/net.openra.OpenRA">Flathub</a>:
  <pre>$ flatpak install --from https://flathub.org/repo/appstream/net.openra.OpenRA.flatpakref
  $ flatpak install --from https://flathub.org/beta-repo/appstream/net.openra.OpenRA.flatpakref</pre>
  The Flatpak sandbox is known to interfere with the mod switching feature between official and community mods.<br />We recommend that players who would like to use this feature run the AppImage releases.
</dd>
```
to match the style for the other packages? The explicit link to FlatHub Beta doesn't seem to add anything unique here.